baybe/searchspace/continuous.py

```diff
@@ -42,6 +44,36 @@ class SubspaceContinuous(SerialMixin):
     )
     """List of linear inequality constraints."""
 
+    def __repr__(self) -> str:
+        """Override the standard __repr__."""
```

Comment:
This docstring is really trivial and says nothing. Can you be more specific in WHAT it actually does (same for the other methods)? If you cannot come up with something useful, you might as well just omit it (provided that our linters allow that, @AVHopp?)

Reply:
Regarding overwriting: Not really sure about that, but since I guess that this will PR will change to overwriting `__str__` instead of `__repr__`, it would be fine for me to skip the docstring there since this is really a well-known method. But we'll have to check that.